README.md

```diff
@@ -0,0 +1,34 @@
+# Ebay Clone Rails App
+Ebay clone that offers a marketplace for buying and selling items.
+
+## Requirement
+- ruby ~> 2.2.1
+- bundler >= 1.3.0, < 2.0
+- rake >= 0.8.7
+- postgresql ~> 0.15
```

Comment:
I'm not sure what this means. I think you mean to say you need PostgreSQL running. Not sure what postgresql version 0.15 means given that PostgreSQL is currently at 9.5 http://www.postgresql.org/